emperor/support_files/js/sceneplotview3d.js

```diff
@@ -100,6 +103,26 @@ define([
     this.dimensionRanges = {'max': [], 'min': []};
     this.drawAxesWithColor(0xFFFFFF);
     this.drawAxesLabelsWithColor(0xFFFFFF);
+
+    this.raycaster = new THREE.Raycaster();
+    this.mouse = new THREE.Vector2();
```

Comment:
The suggestion from [the docs](http://threejs.org/docs/#Reference/Core/Raycaster) and [stackoverflow](http://stackoverflow.com/a/18553739) suggest just instantiating it once and reusing it.